actions/register_org.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: register_org
+runner_type: "python-script"
+pack: github
+description: "Register an organization."
+enabled: true
+entry_point: "retister_org.py"
+parameters:
+  url:
+    type: "string"
+    required: true
+    description: "base API url."
+  github_type:
+    type: "string"
+    required: true
+    description: "type of github: online or enterprise"
+  token:
+    type: "string"
+    required: true
+    description: "API token for the member of the organization"
```

Comment:
minor: technically it's "for **a** member"

Reply:
Thanks. I fix it.